packages/wallet-sdk/src/CoinbaseWalletProvider.test.ts

```diff
@@ -63,7 +62,7 @@ describe('Event handling', () => {
     provider.on('chainChanged', chainChangedListener);
 
     await provider.request({ method: 'eth_requestAccounts' });
-    updateListener.onChainIdUpdate(1);
+    callback('chainChanged', '0x1');
```

Comment:
it needs to be hex string
![image](https://github.com/user-attachments/assets/c012f81f-9708-46ea-a50d-5f6268d5da65)